src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+import java.util.Scanner;
+import task.Deadlines;
+import task.Events;
+import task.Task;
+import task.ToDos;
+
+
+
 public class Duke {
+    public static final String LINE_DIVIDER = "____________________________________________________________";
     public static void main(String[] args) {
+
+        int orderAdded = 1;
+        Task[] recordedTask = new Task[100];
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+
+
         System.out.println("Hello from\n" + logo);
+        printHelloMessage(); //message to welcome users
+
+        Scanner in = new Scanner(System.in);
+
+        while(true) {
+            String commandEntered = in.nextLine(); //awaits user input for the command
+
+
+            if(commandEntered.contains("todo") ||
+                    commandEntered.contains("deadline") ||
+                    commandEntered.contains("event")){
+
+                orderAdded = specificTaskAdder(orderAdded, recordedTask, commandEntered);
+                continue;
+            }
+
+            if(commandEntered.equals("list")){        //command list lists out all recorded words
+                listOfTasksPrinter(orderAdded, recordedTask);
+                continue;
+            }
+            if(commandEntered.contains("done ")){      //command done<space>(number) changes task status of a task from not done to done i.e cross to tick
+                setTaskAsDone(commandEntered, recordedTask);
+                continue;
+            }
+            if(commandEntered.equals("bye")){ //command bye breaks the loop and ends the interaction with Dukebot
+                break;
+            }
+
+            recordedTask[orderAdded-1] = new Task(orderAdded, commandEntered, false);
+            orderAdded = genericTaskAdder(orderAdded, recordedTask, commandEntered);
+
+        }
+        printByeMessage(); //message to bid farewell to users
+    }
+
+    private static int specificTaskAdder(int orderAdded, Task[] recordedTask, String commandEntered) {
+        String[] taskCommandArr = commandEntered.split(" ", 2);
+        String exactDueDate = "";
+        if(taskCommandArr[1].contains("/by")){
```

Comment:
it should have been 
if(taskCommandArr[1].contains("/by")) {

A space after the last ")"

Reply:
made the changes to all!